### PR TITLE
Sidebar Component

### PR DESCRIPTION
**Pull request by ShreyaTumma**
**Overview**
Create a collapsible and expandable sidebar menu component that integrates with all frames. The applications listed in the sidebar menu are added to the menuListItemsConfig list and these applications route to their respective pages when clicked on. 
The page the user is on matches which application is highlighted in the menu's list.

As discussed in the slack channel, I did not style the highlighted application exactly to match the Figma design, as the inverse curve proved difficult to implement. Another task should be opened to address the advanced styling.

**Screenshots**
PRSidebar
PRSidebarExpanded

Closes [OV-58](http://platinum.cscaws.com:8080/browse/OV-58)

### CODE REVIEW

File: cloud-haven-react-app/src/components/Sidebar/Sidebar.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        classNameIcon = "current AppIcon";
+        className = "current current-name rectangle AppName";
+    }else {
+        classNameIcon = "icon-color AppIcon";
+        className = "rectangle AppName";
+    }
+    
+    return(
+
+    <li key={listID} id={listID}>
```
Comment (glkidd) [inline cloud-haven-react-app/src/components/Sidebar/Sidebar.js:29]: Could we indent the whole list item component?

File: cloud-haven-react-app/src/components/Sidebar/Sidebar.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            <button id="Hamburger" onClick={showSidebar}><i className="fas fa-bars">{barIcon}</i></button>
+            <ul>
+                {menuItems}
+            </ul>
+
+        </div>
+    );
+}
+
+export default Sidebar;
```
Comment (glkidd) [inline cloud-haven-react-app/src/components/Sidebar/Sidebar.js:62]: Add a newline at the end of the file

**Review by glkidd — CHANGES_REQUESTED** [review]
Left a couple comments and besides that, the only thing I'd like is if we could maybe center the question mark icon a little better?

**Review by patrickbartlett — COMMENTED** [review]
looks good, just needs centering on the question mark icon

**Review by turrence — APPROVED** [review]

**Review by glkidd — APPROVED** [review]
Looks great! Really good work :)

**Review by patrickbartlett — APPROVED** [review]

**Review by turrence — APPROVED** [review]

**Pull request merged**